src/models/GOKU.jl

```diff
@@ -79,8 +106,19 @@ end
 # Identity by default
 transform_after_diffeq(x, diffeq) = x
 
+@doc raw"""
+    apply_reconstructor(decoder, ẑ)
+Reconstruct the initial data from the extrapolated latent states.
+# Arguments
+decoder: has type Decoder{GOKU} or Decoder{LatentODE}\
+ẑ: Latent states, has size (seq_len,), ẑ[1] has size (latent_dims, batch_size)
+"""
 apply_reconstructor(decoder::Decoder{GOKU}, ẑ) = decoder.reconstructor.(ẑ)
 
+@doc raw"""
+    sample(μ::T, logσ²::T, model::LatentDiffEqModel{GOKU}) where T <: Tuple{Array, Array}
+Samples the parameters and initial state from the normal distribution with mean μ and variance exp(logσ²).
```

Comment:
I would replace "the parameters and initial state" for "latent variables" because these are not even the final initial conditions and parameters.

Reply:
But they are not the latent variables.  How about "abstract representations of the parameters and initial state"?

Reply:
In general in VAE models the variables just after doing the sampling are called "latent variables". Anyway in our case it is  a bit ambiguous because we also talk about the latent space governed by the differential equations.